cli/src/operation/delete_script.rs

```diff
@@ -0,0 +1,33 @@
+use crate::errors::Errors;
+use crate::{CliConfig, SystemId};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct DeleteScript {
+    package_name: String,
+    command: String,
+    delete_instruction: String,
+}
+
+pub async fn delete_script(software_id: &SystemId, config: &CliConfig) -> Result<String, Errors> {
+    let script_elements = reqwest::Client::new()
+        .delete(format!(
+            "{}/software/{}/system/{}",
```

Comment:
I guess, there's a `Url::join` for that.

It can result in an error, meaning that database url provided in a config cannot be a base url (rn this will result in http error, I guess).